test/test-parse-expression-statement.cpp

```diff
@@ -769,7 +769,7 @@ TEST(test_parse, expression_statement) {
   }
 
   {
-    spy_visitor v = parse_and_visit_statement(u8"async => rhs;"_sv);
+    spy_visitor v = parse_and_visit_statement(u8"(async => rhs)"_sv);
```

Comment:
Must fix: This test is trying to test the statement-starts-with-`async` code path. Adding parentheses here reduces test coverage. Undo this change.